### ISSUE TITLE
Evaluate and adapt site to fit color contrast accessibility guidelines

### ISSUE THREAD

**Issue by k-y-le** [Representable/representable#414]
**Is your feature request related to a problem? Please describe.**
Currently, some parts of the site fail accessibility checks because the default 'primary' color we use with bootstrap is below the 4.5:1 color contrast standard for accessibility.

**Describe the solution you'd like**
Colors changed to have a higher contrast - on all pages, especially need to look over About page
